CHANGELOG.markdown

```diff
@@ -1,3 +1,7 @@
+6.?????
```

Comment:
I think it would suffice to only bump the secondary version number (e.g., `5.5.?????`), since this doesn't break any other code in the Kmettiverse AFAICT, and that's the usual metric by which "Kmett-major" version bumps are measured.